docs/data/joy/getting-started/templates/rental-dashboard/App.tsx

```diff
@@ -0,0 +1,135 @@
+        <Header />
+        <FirstSidebar />
+        <Main>
+          <Grid
+            container
+            sx={{
+              width: '100%',
+              height: '100dvh',
+
+              margin: 0,
+            }}
+          >
+            <Grid
+              xs={12}
+              lg={8}
+              sx={{
+                overflowY: 'scroll',
+                height: '100%',
+                px: { xs: 2, md: 4 },
+                pt: { xs: 8, md: 4 },
+                pb: 5,
+              }}
+            >
+              <Stack spacing={2}>
+                <HeaderSection />
+                <Divider />
+                <Box
+                  sx={{
+                    width: '100%',
+                    height: 360,
+                    backgroundSize: 'cover',
+                    backgroundImage:
+                      'url("https://images.unsplash.com/photo-1581922819941-6ab31ab79afc?auto=format&fit=crop&w=700")',
+                  }}
+                  display={{ xs: 'block', md: 'none' }}
+                />
+                <Filters />
+                <Search />
+                <Toggles />
+                <RentalCard
+                  title="A Stylish Apt, 5 min walk to Queen Victoria Market"
+                  category="Entire apartment rental in Collingwood"
+                  rareFind
+                  image="https://images.unsplash.com/photo-1568605114967-8130f3a36994?auto=format&fit=crop&w=400"
+                />
+                <RentalCard
+                  title="Designer NY style loft"
+                  category="Entire loft in central business district"
+                  liked
+                  image="https://images.unsplash.com/photo-1600596542815-ffad4c1539a9?auto=format&fit=crop&w=400"
+                />
+                <RentalCard
+                  title="5 minute walk from University of Melbourne"
+                  category="Entire rental unit in Carlton"
+                  image="https://images.unsplash.com/photo-1537726235470-8504e3beef77?auto=format&fit=crop&w=400"
+                />
+                <RentalCard
+                  title="Magnificent apartment next to public transport"
+                  category="Entire apartment rental in Collingwood"
+                  image="https://images.unsplash.com/photo-1618221195710-dd6b41faaea6?auto=format&fit=crop&w=400"
+                />
+                <Divider />
+                <Pagination />
+              </Stack>
+            </Grid>
+            <Grid xs={0} lg={4}>
+              <Stack direction="row">
+                <Divider orientation="vertical" sx={{ height: '100dvh' }} />
+                <Box
+                  sx={{
+                    width: '100%',
+                    m: 1.5,
+                    backgroundSize: 'cover',
+                    backgroundImage:
+                      'url("https://images.unsplash.com/photo-1581922819941-6ab31ab79afc?auto=format&fit=crop&w=700")',
```

Comment:
```suggestion
                    width: '100%',
                    m: 1,
                    borderRadius: 'sm',
                    backgroundSize: 'cover',
                    backgroundImage:
                      'url("https://images.unsplash.com/photo-1478860409698-8707f313ee8b?ixlib=rb-4.0.3&ixid=M3wxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8fA%3D%3D&auto=format&fit=crop&w=4000&q=80")',
```

Just a few tiny styling tweaks to the image + swapping it for another that's a bit less abstract and higher quality!

Reply:
Thank you for fixing a bug. 🎉